src/dmd/backend/elem.d

```diff
@@ -444,13 +533,17 @@ void el_copy(elem *to,elem *from)
 
 elem * el_alloctmp(tym_t ty)
 {
-  symbol *s;
+    version (MARS)
+    { }
+    else
+        assert(!PARSER);
```

Comment:
The formatting you used later: `version (MARS) { } else assert(!PARSER);` looks more natural.